angular/core/components/thread_page/previous_proposals_card/previous_proposals_card.coffee

```diff
@@ -20,7 +20,7 @@ angular.module('loomioApp').directive 'previousProposalsCard', ->
     setLastClosedProposal = ->
       return unless $scope.anyProposals() and !$scope.discussion.hasActiveProposal()
       proposal = $scope.discussion.closedProposals()[0]
-      proposal if moment().add(-1, 'month') < proposal.closedAt
+      proposal.id if moment().add(-1, 'month') < proposal.closedAt
```

Comment:
Ah, hm, so this was the intended behaviour all along, we were just returning the wrong thing.

Any idea about why those few tests are failing?